graphql/mutation_resolver.go

```diff
@@ -472,14 +472,17 @@ func (r *mutationResolver) UnschedulePatchTasks(ctx context.Context, patchID str
 }
 
 // AddFavoriteProject is the resolver for the addFavoriteProject field.
-func (r *mutationResolver) AddFavoriteProject(ctx context.Context, identifier string) (*restModel.APIProjectRef, error) {
-	p, err := model.FindBranchProjectRef(identifier)
+func (r *mutationResolver) AddFavoriteProject(ctx context.Context, identifier *string, projectIdentifier *string) (*restModel.APIProjectRef, error) {
+	if projectIdentifier == nil {
+		projectIdentifier = identifier
+	}
+	p, err := model.FindBranchProjectRef(utility.FromStringPtr(projectIdentifier))
 	if err != nil || p == nil {
-		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("could not find project '%s'", identifier))
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("could not find project '%s'", utility.FromStringPtr(projectIdentifier)))
 	}
 
 	usr := mustHaveUser(ctx)
-	err = usr.AddFavoritedProject(identifier)
+	err = usr.AddFavoritedProject(utility.FromStringPtr(identifier))
```

Comment:
I think you want it to be:
```suggestion
	err = usr.AddFavoritedProject(utility.FromStringPtr(projectIdentifier))
```